src/Pyz/Zed/Console/ConsoleDependencyProvider.php

```diff
@@ -424,7 +423,6 @@ protected function getConsoleCommands(Container $container): array
             $commands[] = new RemoveGlueIdeAutoCompletionConsole();
             $commands[] = new DataBuilderGeneratorConsole();
             $commands[] = new RemoveDataBuilderConsole();
-            $commands[] = new CompletionCommand();
             $commands[] = new DataBuilderGeneratorConsole();
             $commands[] = new PropelSchemaValidatorConsole();
```

Comment:
Why we do not have `ReservationVersionPostSaveTerminationAwareStrategyPlugin` related change, as in other PRs?

Reply:
because it is already present 
https://github.com/spryker-shop/b2c-demo-marketplace/blob/internal/src/Pyz/Zed/Oms/OmsDependencyProvider.php#L177